sample/confidential_client_certificate_sample.py

```diff
@@ -53,10 +56,11 @@
     result = app.acquire_token_for_client(scopes=config["scope"])
 
 if "access_token" in result:
-    print(result["access_token"])
-    print(result["token_type"])
-    print(result["expires_in"])  # You don't normally need to care about this.
-                                 # It will be good for at least 5 minutes.
+    # Calling graph using the access token
+    graph_data = requests.get(  # Use token to call downstream service
+        config["endpoint"],
+        headers={'Authorization': 'Bearer ' + result['access_token']}, ).json()
+    print("Users from graph: " + str(graph_data))
```

Comment:
Nit: Given that we make the `endpoint` configurable in settings, it would seem more appropriate we also use a more generic term here, perhaps something like `print("Graph API call result: %s" % graph_data)`?